react/Viewer/ImageLoader.jsx

```diff
@@ -16,6 +15,10 @@ class ImageLoader extends React.Component {
     src: null
   }
 
+  static contextTypes = {
+    client: PropTypes.object.isRequired
+  }
```

Comment:
We stopped using the old context API, we should not use it again. HOC is fine. Why this change? Does the code was not working with withClient but now works with contextType ? 🤔

Reply:
Oui c'est ça. Avec `withClient` client est `null`, donc pas possible de faire un `client.something`. J'ai pris exemple sur `withFileUrl.jsx` qui fonctionne comme ça.

Reply:
Il faut utiliser `<CozyProvider client={mockClient}>` dans le DemoProvider, et ça devrait le faire.

Reply:
Ok top merci pour l'astuce je regarde ça